addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -362,7 +318,11 @@ export class AttachmentViewer extends Component {
         }
         // TODO: clicking on the background should probably be handled by the dialog?
         // task-2092965
-        this._close();
+        if (!isEventHandled(ev, 'attachmentViewer.clickNext') &&
+            !isEventHandled(ev, 'attachmentViewer.clickPrevious')
+        ) {
```

Comment:
current guideline for multiline condition is to go to new line also for the first one

Reply:
also just inverse the conditions add them to quick return instead of adding indent level for the rest of the code